examples/computer_vision_techniques/mask_disk.py

```diff
@@ -41,7 +42,7 @@
 mask = ma.masked_less_equal(r, 1)
 palette = aia.plot_settings['cmap']
 palette.set_bad('black')
-
+norm = mpl.colors.Normalize(vmin=-1000, vmax=3000)
```

Comment:
Can you add a comment here as to why we had to add this line?

Reply:
i have added comment, i have tried for this image only for others i think we have to experiment with different values

Reply:
That's a problem. We can't expect our users to have to manually change the norm in some (what feels like random) way to get the mask to work. It's ok for this example but I don't think we can count it as a fix for the original issue.